blob/azureblob/azureblob_test.go

```diff
@@ -32,15 +32,17 @@ import (
 	"gocloud.dev/internal/testing/setup"
 )
 
-// Prerequisites for --record mode
+// Prerequisites for -record mode
 // 1. Sign-in to your Azure Subscription at http://portal.azure.com.
 //
 // 2. Create a Storage Account.
 //
 // 3. Locate the Access Key (Primary or Secondary) under your Storage Account > Settings > Access Keys.
 //
-// 4. Set the environment variables AZURE_STORAGE_ACCOUNT, AZURE_STORAGE_KEY to
-//    the storage account name and your access key.
+// 4. Set the environment variables AZURE_STORAGE_ACCOUNT, AZURE_STORAGE_KEY, and AZURE_CLOUD_ENVIRONMENT to
```

Comment:
I think you can leave this unchanged; we're not going to `-record` in anything other than AzureCloud so we can just leave it empty.